src/rmview/rmview.py

```diff
@@ -507,11 +540,28 @@ def frameError(self, e):
     QMessageBox.critical(self.viewer, "Error", 'Please check your reMarkable is properly configured, see the documentation for instructions.\n\n%s' % e)
     self.quit()
 
+  def event(self, e):
+    return QApplication.event(self, e)
+
 def rmViewMain():
   log.setLevel(logging.INFO)
+  if len(sys.argv) > 1:
```

Comment:
Yeah, that works for now, but in the future it would be better and more robust to utilize ``argparse`` module :)